packages/plugin-core/src/views/DendronTreeViewV2.ts

```diff
@@ -9,41 +9,57 @@ import {
   VSCodeEvents,
 } from "@dendronhq/common-all";
 import { getDurationMilliseconds } from "@dendronhq/common-server";
-import { WorkspaceUtils } from "@dendronhq/engine-server";
+import { EngineEventEmitter, WorkspaceUtils } from "@dendronhq/engine-server";
 import _ from "lodash";
 import path from "path";
 import * as vscode from "vscode";
+import { Disposable } from "vscode-languageclient";
```

Comment:
This is a bit confusing, but this is the technically the wrong Disposable (although functionally it _probably_ doesn't make a diff). 

`import { Disposable } from "vscode";`

We should do a quick sweep for other places that import it from `vscode-languageclient`, I see that there are several.

Reply:
sure fixed